doc-content/enterprise-only/migration/migration-tool-niogit-7.4-proc.adoc

```diff
@@ -0,0 +1,217 @@
+      "releasesEnabled":true,
+      "releasesUpdatePolicy":"never",
+      "snapshotEnabled":true,
+      "snapshotUpdatePolicy":"never"
+    },
+    {
+      "id":"kie-internal-staging-repository-group",
+      "name":"KIE Internal Staging Repositories",
+      "url":"https://origin-repository.jboss.org/nexus/content/groups/kie-internal-group",
+      "releasesEnabled":true,
+      "releasesUpdatePolicy":"never",
+      "snapshotEnabled":false,
+      "snapshotUpdatePolicy":"never"
+    }
+
+  ],
+  "pluginRepositories-remove":[],
+  "pluginRepositories-update-urls":[]
+}
+----
+
+Example `pom-migration.json` file with `dependencies` only:
+
+[source,json]
+----
+{
+  "dependencies":[
+
+    {"groupId":"junit", "artifactId":"junit", "version":"4.12", "scope":"test"},
+    {"groupId":"com.thoughtworks.xstream", "artifactId":"xstream", "version":"4.12", "scope":"test"}
+
+  ],
+  "repositories-add":[],
+  "repositories-remove":[],
+  "repositories-update-urls":[],
+  "pluginRepositories-add":[],
+  "pluginRepositories-remove":[],
+  "pluginRepositories-update-urls":[]
+}
+----
+--
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+* *Product:* {PRODUCT_SHORT}
+* *Version:* 7.5
+
+. Download *{PRODUCT} 7.5 Add-Ons* and extract the downloaded `{PRODUCT_INIT}-7.5.0-add-ons.zip` file to a temporary directory.
+. In the extracted `{PRODUCT_FILE}-add-ons` folder, extract the `{PRODUCT_INIT}-7.5-migration-tool.zip` sub-folder. The migration tool is in the `bin` directory.
+. In your command terminal, navigate to the temporary directory where you extracted the `{PRODUCT_INIT}-7.5-migration-tool` folder and run the migration tool. The `${PRODUCT_INIT_CAP}_NIOGIT_DIR` portion is the path to the `.niogit` directory that you previously copied to the {PRODUCT} 7.5 installation.
++
+--
+On Linux or UNIX-based systems:
+[source,subs="attributes+"]
+----
+$ cd $INSTALL_DIR/{PRODUCT_INIT}-7.5-migration-tool/bin
+$ ./migration-tool.sh -t ${PRODUCT_INIT_CAP}_NIOGIT_DIR
+----
+
+On Windows:
+[source,subs="attributes+"]
+ifdef::DM[]
+----
+$ cd $INSTALL_DIR\rhdm-7.5-migration-tool/bin
+$ migration-tool.bat -t ${PRODUCT_INIT_CAP}_NIOGIT_DIR
+----
+endif::DM[]
+ifdef::PAM[]
+----
+$ cd $INSTALL_DIR\rhpam-7.5-migration-tool\bin
+$ migration-tool.bat -t ${PRODUCT_INIT_CAP}_NIOGIT_DIR
+----
+endif::PAM[]
+
+In the command prompt that appears, the following options are displayed:
+
+* *Project structure migration*: Migrates the {PRODUCT} 7.4 project repository structure to the new project-oriented structure used in {PRODUCT} {PRODUCT_VERSION}.
+* *System configuration directory structure migration*: Migrates the `system.git` repository structure used in {PRODUCT} 7.4 to the new structure used in {PRODUCT} 7.5. This migration option requires the project structure migration to be executed first.
```

Comment:
I would rephrase to avoid passive voice, something like "You must complete the project structure migration before selecting this option"